bin/setup

```diff
@@ -10,13 +10,13 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
-# Set up configurable environment variables
-if [ ! -f .env ]; then
-  cp .sample.env .env
-fi
-
 # Set up database and add any development seed data
-bundle exec rake db:setup dev:prime
+bundle exec rake db:setup
```

Comment:
I hadn't noticed this `dev:prime` task that Suspenders provides. I think the entries I added to `seeds.db` should actually have been added here instead. I'll open a PR to move them.